app/src/main/java/com/example/kakaowebtoon/data/mapper/DataMapper.kt

```diff
@@ -1,10 +1,23 @@
 package com.example.kakaowebtoon.data.mapper
 
+import com.example.kakaowebtoon.data.dataremote.model.response.DailyWebtoon
 import com.example.kakaowebtoon.data.dataremote.model.response.DummyResponseDto
 import com.example.kakaowebtoon.domain.model.DummyData
+import com.example.kakaowebtoon.domain.model.WebtoonCard
 
 fun DummyResponseDto.toDomainModel(): DummyData {
     return DummyData(
         description = this.dummy + "입니다 "
     )
 }
+
+fun DailyWebtoon.toDomainModel(): WebtoonCard {
+    return WebtoonCard(
+        id = id,
+        imageUrl = image,
+        title = title,
+        promotion = "",
+        author = "",
+        genre = ""
+    )
+}
```

Comment:
```suggestion
fun DailyWebtoon.toDomainModel(): WebtoonCard 
    =WebtoonCard(
        id = this.id,
        imageUrl = this.image,
        title = this.title,
        promotion = this.promotion?:"",
        author = this.author?:"",
        genre = this.genre?:""
    )

```
이렇게 쓰는거랑 기능적으론 차이가 없는거죠?
그리고 프로모션, author,genre는 nullable인가용가리?

Reply:
네 맞습니다!!!
요일별 웹툰 API에서는 아예 해당 필드를 받지 않아서 필요가 없는데 id, url, title 동일한 필드를 가지는 데이터 클래스를 또 생성할까 말까 고민하다 일단은 이렇게 해뒀습니다!!